docs/API.md

```diff
@@ -371,7 +371,7 @@ $ curl -X POST --header "Content-Type:application/json" \
 Temporarily stops a user application container. A reboot or supervisor restart will cause the container to start again.
 The container is not removed with this endpoint.
 
-This is only supported on single-container devices, and will return 400 on devices running multiple containers.
+This is only supported on single-container devices, and will return 400 on devices running multiple containers. Refer to v2 endpoint, [`/v2/applications/:appId:/stop-service`](#post-v2applicationsappidstop-service) for running the query on multiple containers.
```

Comment:
@vipulgupta2048  At the risk of giving you one more thing to do, there is an extra colon after appId which is definitely a typo :D (Same goes for the `/v2/applications/:appId/stop-service` entry in this documentation)